services/pane.js

```diff
@@ -156,17 +181,31 @@ function addPreview(preview) {
   }
 }
 
+/**
+ * format and assemble error messages
+ * @param {Object[]} errors
+ * @returns {Element}
+ */
 function addErrors(errors) {
   return _.reduce(errors, function (el, error) {
-    var errorEl = dom.create(`
-        <div class="publish-error">
-          <span class="label">${error.rule.label}:</span>
-          <span class="description">${error.rule.description}</span>
-          <ul class="errors"></ul>
-        </div>
-      `),
+    var errorEl = dom.find('.publish-errors'),
+      errorLabel = dom.find(errorEl, '.label'),
+      errorDescription = dom.find(errorEl, '.description'),
       list = dom.find(errorEl, 'ul');
 
+    // reset template label default if available
```

Comment:
The label value should either be handled in the template or in the js. If the template needs it for readability, could use an HTML or nunjucks comment.